test/images/agnhost/agnhost.go

```diff
@@ -51,7 +51,7 @@ import (
 func main() {
 	rootCmd := &cobra.Command{
 		Use:     "app",
-		Version: "2.22",
+		Version: "2.23",
```

Comment:
Why do we need to update the version in 3 places?  Can't we inject it at build time via linker?

Reply:
I agree that it is not ok to have such kind of duplication, but I do not implement it here. I just do minimal changes needed to fix the issue.

Reply:
sure, that's fine - I don't know who owns this sub-component now, so I guess I hope they see this and agree.